registry/repository.go

```diff
@@ -100,8 +84,28 @@ type ReferrerFinder interface {
 	Referrers(ctx context.Context, desc ocispec.Descriptor, artifactType string, fn func(referrers []ocispec.Descriptor) error) error
 }
 
+// TagFinder discovers tags by the tag service.
+type TagFinder interface {
```

Comment:
Why this interface is added to `registry` package but not `content`?

Reply:
How about calling it `TagLister`? 🤔
`ReferrerFinder` is to find the referrers of a descirptor.
`TagLister` is to list tags in the repository.
If it is called `TagFinder` , it sounds like it should find tags for a descriptor?

Reply:
If we call it `TagLister`, we should re-name `ReferrerFinder` to `ReferrerLister` since it is [listing referrers](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#listing-referrers).